Create data infra in a box diab billy

billyteo opened the pull request.

candyyap-gvt reviewed (commented):
**Overview**
1. I think we can remove "previously known as im8"
2. There is a blog on IaC. Should we link to it? https://www.developer.tech.gov.sg/2021/07/22/security-benefits-iac.html
3. The summary para feels like a repeat of para 3

**Key Benefits**
1. Para 2 si similar to on of the key benefits. If it is in key benefits, i think we can remove para 2 to keep the content succinct

**Features**
1. Feature 1: Replace the acronym IM with the correct policy name. I think is also known as (ICT&SS) which we can include the acronym in the overview page
2. Feature 2: For most of the writeup, we hyperlink the product name instead of adding additional sentence to ask peopl to refer here. Should we standardise it?
3. Feature 3: Is there a need to put in (UAT) since it is not used anywhere else?

**Use cases**
As of now, i only know how to deploy DIAB but fig 4 shows a use case so how do i use the reference architecture template? Are we publishing any steps anywhere? Under features, reference architecture, it also states that DIAB provides various use cases. What are the use cases?

**Getting Started**
1. Can we link Owns a SHIP-HATS account to ship-hats documentation on doc svc on how to create account?
2. Where is DIAB Terms of Use?
3. Can we publish the onboarding form and in the form, to inseret the email for them to submit the form?

**Resources**
Under tech documentation, shouldn't we point SIaC to the tech content rather than the product writeup?

Small matter but do we need to keep repeating DIAB in key benefits and features?

billyteo commented:
> **Overview**
> 
> 1. I think we can remove "previously known as im8"
Ok,removed!
> 2. There is a blog on IaC. Should we link to it? https://www.developer.tech.gov.sg/2021/07/22/security-benefits-iac.html
Ok, linked!
> 3. The summary para feels like a repeat of para 3
Ok, have edited!

> 
> **Key Benefits**
> 
> 1. Para 2 si similar to on of the key benefits. If it is in key benefits, i think we can remove para 2 to keep the content succinct
Ok, tweaked and edited!
> 
> **Features**
> 
> 1. Feature 1: Replace the acronym IM with the correct policy name. I think is also known as (ICT&SS) which we can include the acronym in the overview page
Ok, done!
> 2. Feature 2: For most of the writeup, we hyperlink the product name instead of adding additional sentence to ask peopl to refer here. Should we standardise it?
Ok, have standardised.
> 3. Feature 3: Is there a need to put in (UAT) since it is not used anywhere else?
Ok, have removed (UAT).
> 
> **Use cases** As of now, i only know how to deploy DIAB but fig 4 shows a use case so how do i use the reference architecture template? Are we publishing any steps anywhere? Under features, reference architecture, it also states that DIAB provides various use cases. What are the use cases?
> 

Have removed for now, and will work with Daniel to put in more information to guide agencies. Will also set up meeting with him to discuss this.

> **Getting Started**
> 
> 1. Can we link Owns a SHIP-HATS account to ship-hats documentation on doc svc on how to create account?

Done!

> 2. Where is DIAB Terms of Use?
> 3. Can we publish the onboarding form and in the form, to inseret the email for them to submit the form?
> 

Will work with Daniel on 2 and 3.

> **Resources** Under tech documentation, shouldn't we point SIaC to the tech content rather than the product writeup?
> 

Removed SIaC under Tech Documentation as discussed.

> Small matter but do we need to keep repeating DIAB in key benefits and features?

Have edited and tweaked the writing.
